### ISSUE TITLE
toTitleCase does not work

### ISSUE THREAD

**Issue by solsticedhiver** [mrgnhnt96/change_case#1]
If I am not wrong, `.toTitleCase()` does not change the case at all

Example:

    "WE'RE ALMOST HERE".toTitlelCase() // "WE'RE ALMOST HERE."

**Comment by aescobar-rootstrap** [comment]
> If I am not wrong, `.toTitleCase()` does not change the case at all
> 
> Example:
> 
> ```
> "WE'RE ALMOST HERE".toTitlelCase() // "WE'RE ALMOST HERE."
> ```

Might wanna check if your font allows lowercase letters. Or maybe there's some global override in your code.

**Comment by solsticedhiver** [comment]
Here is a simple dart code that shows the bug:
```dart
import 'package:change_case/change_case.dart';

void main() {
        print('Test of change_case module'.toTitleCase());
        print('TEST of change_case MODULE'.toTitleCase());
}
/* OUTPUT
Test of Change_case Module
TEST of Change_case MODULE
*/
```

I would have expected the same string twice.

The "bug" is easy to find: line 29 of change_case/lib/src/cases/title.dart, you are simply ignoring already capitalize word.

That is your choice, may be.

So I guess this is not an issue, should be closed as "will not fix" ?

**Comment by mrgnhnt96** [comment]
Yeah, I have come across the behavior has well. I'll take a look at it 👍

**Comment by mrgnhnt96** [comment]
This fix is released in [v2.2.0](https://pub.dev/packages/change_case/versions/2.2.0)
